commit b54303bc9b9310e288a1dde9c923f48e854bd55f
Author: mean <fixounet@free.fr>
Date:   Tue Jul 5 18:43:58 2016 +0200

    [Prefs/couple] Add std::string in addition to char *, safer

diff --git a/avidemux_core/ADM_coreUtils/src/prefs.cpp b/avidemux_core/ADM_coreUtils/src/prefs.cpp
index 79f69c1..7c1b419 100644
--- a/avidemux_core/ADM_coreUtils/src/prefs.cpp
+++ b/avidemux_core/ADM_coreUtils/src/prefs.cpp
@@ -44,8 +44,8 @@ typedef struct
    const char *name2;
    ADM_paramType  type;
    const char *defaultValue;
-   float    min;
-   float    max;
+   double    min;
+   double    max;
 }optionDesc;

 #include "prefs2_pref.h"


